configuration/pih/htmlforms/hiv/section-hiv-state.xml

```diff
@@ -934,9 +934,9 @@
 
             <div class="section-container">
                 <p>
-                    <obs conceptId="CIEL:164801" style="checkbox"
+                    <obs id="syph-checkbox" conceptId="CIEL:164801" style="checkbox"
```

Comment:
I added an id in so that we can access it via the HTML Form Entry helper functoins, see below...